markdown/assignments/pool_b/xp/index.md

```diff
@@ -2,15 +2,26 @@
 type: assignment
 title: "Erfahrungspunkte"
 author: "Andre Matutat (FH Bielefeld)"
-points:
-weight: 1
+points: 2
+weight: 6
 hidden: true
 ---
 
+## Ziel
+
+In dieser Aufgabe implementieren Sie Erfahrungspunkte, die der Spieler sammeln kann, um im Level aufzusteigen.
```

Comment:
Was bedeutet "im Level aufsteigen"? Hier geht es ja darum, ins nächste Level _abzusteigen_ ;) ... 

Ich würde den Nachsatz einfach streichen und eine Ergänzung vornehmen: "Mit mehr Erfahrung gewinnt der Spieler häufiger Kämpfe oder wird nicht so schnell hungrig oder ..."

Reply:
Umbenennen in (Helden)stufe?
Dann gibt es keine Doppelbelegung des Begriffes "Level"